doc/osh-help.md

```diff
@@ -805,6 +806,135 @@ these are discouraged.
 
 <!--    -R VAR     True if the variable VAR has been set and is a nameref variable. -->
 
+<a h4="printf">printf</h4>
+
+    printf FLAG? FMT ARGS*
```

Comment:
Even though there's only one flag, I think `FLAG*` is more correct, since

`printf -v one -v two %s arg`

 is allowed, and that is true for all builtins.  The flag parsers prefer the last value.